src/platform/plugins/shared/dashboard/public/dashboard_api/layout_manager/layout_manager.ts

```diff
@@ -130,7 +156,8 @@ export function initializeLayoutManager(
   const placeNewPanel = async (
     uuid: string,
     panelPackage: PanelPackage,
-    grid?: DashboardPanel['grid']
+    grid?: DashboardPanel['grid'],
+    beside?: string
```

Comment:
`beside` allows ES|QL control panels to be placed beside their relevant ES|QL panels:

https://github.com/user-attachments/assets/823ff0e7-d44e-42bb-b6fc-1873a7de9550

Notice that the new control is not currently focused. This will be addressed in a [follow up](https://github.com/elastic/kibana/issues/236021).